plugin/src/main/scala/org/scalafmt/sbt/ScalafmtPlugin.scala

```diff
@@ -233,13 +267,21 @@ object ScalafmtPlugin extends AutoPlugin {
   }
 
   lazy val scalafmtConfigSettings: Seq[Def.Setting[_]] = Seq(
-    scalafmt := formatSources(
-      streams.value.cacheDirectory,
-      (unmanagedSources in scalafmt).value,
-      scalaConfig.value,
-      streams.value.log,
-      streams.value.text()
-    ),
+    scalafmt := {
+      scalafmtImpl.value
+      ()
+    },
+    scalafmtImpl := {
```

Comment:
Is there any way to exclude `*Impl` tasks from public api?

Reply:
Because it's not part of the autoImport, it won't be available to build.sbt.
@eatkins suggested using `_` prefix in his plugin to hide the key, I think.

Reply:
Could you not define these with `Initialize` values defined with `Def.task` and replace:

```scala
(scalafmtImpl / streams).value.cacheStoreFactory
```

with the downstream

```scala
(scalafmt / streams).value.cacheStoreFactory
```
?

Reply:
@dwijnand Yea. Now that I'm not using the previous value, maybe I can do that.